### PR TITLE
User guide: Update the section on reading math to recommend MathCAT, and also clarify some information about MathPlayer.

### LINKED ISSUE #15352: Update the user guide section 7 to include Mathcat instead of Mathplayer
### Is your feature request related to a problem? 
The section 7 in the user guide is outdated. MathPlayer has been discontinued and Mathcat is the follow-up project. This should be described and updated in the user guide.

### Describe the solution you'd like
Updated information in the user guide.

### Describe alternatives you've considered
None

### Additional context
None

### LINKED ISSUE #16036: Documentation is inadequate for math support setup
### Steps to reproduce:

1. Read [NVDA documentation for adding math support](https://www.nvaccess.org/files/nvda/documentation/userGuide.html#ReadingMath).
2. Follow the link as instructed: https://www.dessci.com/en/products/mathplayer/
3. Observe that the browser redirects to: https://info.wiris.com/mathplayer-info
4. Read the documentation on the Wiris web page, which NVDA depends on by reference.
5. Determine whether it is necessary to install MathPlayer 4 based on the instructions.

### Actual behavior:

NVDA documentation says:

> Using MathPlayer 4 from Design Science, NVDA can read and interactively navigate supported mathematical content. This requires that MathPlayer 4 is installed on the computer. MathPlayer is available as a free download from: https://www.dessci.com/en/products/mathplayer/. After installing MathPlayer, restart NVDA. 

The Wiris documentation (which NVDA depends on) says:

> MathPlayer views and speaks web pages containing math. Accessibility for web environments using MathPlayer is limited to obsolete browsers such as Internet Explorer 8.

Taken together, the above NVDA and Wiris instructions currently inform users that MathPlayer is necessary, but only for Internet Explorer 8. However, this is incorrect. For example, if I perform the "Steps to reproduce" in #16035 using Firefox or Chrome, without first installing MathPlayer, then NVDA does not announce the structure of MathML content.

### Expected behavior:

If MathPlayer 4 is required for all browsers, instructions should clearly say that it is required.

### NVDA logs, crash dumps and other attachments:

not checked

#### NVDA installed/portable/running from source:

installed

#### NVDA version:

Version: 2023.2 (2023.2.0.29051)

#### Windows version:

Version 10.0.19045 Build 19045

#### Name and version of other software in use when reproducing the issue:

Chrome 120
Firefox 121.0.1

#### Other information about your system:

none

#### Does the issue still occur after restarting your computer?

yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

no

#### If NVDA add-ons are disabled, is your problem still occurring?

not attempted

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

not attempted

### PR DESCRIPTION
**Pull request by michaelDCurran**
### Link to issue number:
Fixes #16036
Fixes #15352 

### Summary of the issue:
When reading the section of the User guide about Math support, and then the MathPlayer page from Wiris, it is unclear as to whether MathPlayer is actually needed for NvDA unless you are using an older browser such as Internet Explorer 8.
Also, Nthe User Guide does not recommend let alone mention, MathCAT, which is now the replacement for the older and unmaintained MathPlayer.

### Description of user facing changes
Update the math section in the User guide to mention and recommend MathCAT, and clarify that MathPlayer  is still relevant on newer browsers. 

### Description of development approach
n/a

### Testing strategy:
Built the user guide as html.

### Known issues with pull request:
None known.

### Code Review Checklist:

- [x] Documentation:
  - Change log entry
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
- [x] Testing:
  - Unit tests
  - System (end to end) tests
  - Manual testing
- [x] UX of all users considered:
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English
- [x] API is compatible with existing add-ons.
- [x] Security precautions taken.

### CODE REVIEW

**Comment by nvdaes** [comment]
@michaelDCurran , I don't see a mention to Neil to notify about this PR.
I love MathCAT and when needed I help to make smoke testing and to submit the add-on to the store if Neil ask me to do it, so let's mention @nsoiffer
I'll strongly recommend MathCAT for languages like spanish, since CMU braille is supported and used in certain countries, and I found difficulties to navigate with MathPlayer.

**Comment by CyrilleB79** [comment]
These changes generally look good to me.

I have just one point to discuss.
Only MathCAT and MathPlayer are mentioned as a possibility to access math content with NVDA. Though, using Access8Math is another real-life and quite popular alternative. It is specifically relevant for languages that are not (yet) available in MathCAT: fr, pt_BR, ru, tr, as well as zh_CN and zh_TW (considering that zh without country code is available in MathCAT instead).

It seems that NV Access wants to promote MathCAT and not Access8Math. Could you list in this issue (not in the User Guide) all the reasons for this choice and if it is likely to change in the future or not? This would encourage translators to put effort where it is more useful. Thanks.

Even if MathCAT is recommended in the User Guide, I think that mentioning Access8Math too would be useful, indicating that there are different supported languages with respect to MathCAT. Thanks.

**Comment by Qchristensen** [comment]
Changes look good so far.  One small change: Line 932 reads: "this is only referring to using mathPlayer to display mathematical content visually, and can be ignored by those using it to read or navigate math with NvDA." - there should be a capital at the start of the line and the "v" in NVDA should be capitalised.

@CyrilleB79 Not speaking for @michaelDCurran but I suspect the original intention was to encourage use of MathCat over Mathplayer, rather than specifically excluding Access8Math.  Would you like to propose an amended wording to include Access8Math for the use cases it is best suited to?

**Comment by michaelDCurran** [comment]
@CyrilleB79 To be completely honest, I was completely unaware of Access8math. But having now tested it a bit, it is definitely worthy of being included in the user guide. Thus I have rewritten the section to mention both MathCAT and Access8math. And to Generally recommend one of those newer NVDA add-ons, over the older MathPlayer, rather than the section generally recommending MathCAT specifically. Does this read better now?

**Comment by CyrilleB79** [comment]
> @CyrilleB79 To be completely honest, I was completely unaware of Access8math. But having now tested it a bit, it is definitely worthy of being included in the user guide. Thus I have rewritten the section to mention both MathCAT and Access8math. And to Generally recommend one of those newer NVDA add-ons, over the older MathPlayer, rather than the section generally recommending MathCAT specifically. Does this read better now?

Yes, very nice. Thanks.

I have one last question: The URL for Access8Math is https://addons.nvda-project.org/addons/access8math.en.html.
For now the community website cannot be updated due to translation system being paused. Moreover,
Additionally, in the medium term, this website may be removed; some add-on authors have already changed the documentation URL to their GitHub repository.

@michaelDCurran, any update regarding the community website? Can add-on authors rely on it for NVDA 2024.1? Or do you already recommend to use an alternative URL?

@Tseng Woody, do you plann to change the documentation URL for the next release of Access8Math? You may wait for Mick's answer to decide.

**Comment by michaelDCurran** [comment]
I would recommend that add-on authors move their home page URLs away 
from the old add-on website at this stage.

File: user_docs/en/userGuide.t2t [hunk 1]
```diff
@@ -915,12 +915,14 @@ Also, although the application will try its best to match the native selection t
 However, for scenarios where you wish to copy an entire table or paragraph of rich content, this feature should prove useful.
  
 + Reading Mathematical Content +[ReadingMath]
-Using MathPlayer 4 from Design Science, NVDA can read and interactively navigate supported mathematical content.
-This requires that MathPlayer 4 is installed on the computer.
-MathPlayer is available as a free download from the [MathPlayer information page https://info.wiris.com/mathplayer-info].
-After installing MathPlayer, restart NVDA.
-
-NVDA supports the following types of mathematical content:
+NVDA can  read and navigate mathematical content on the web and in other applications, providing access in both speech and braille.
```
Comment (seanbudd) [inline user_docs/en/userGuide.t2t:918]: remove extra space

```suggestion
NVDA can read and navigate mathematical content on the web and in other applications, providing access in both speech and braille. 
```

File: user_docs/en/userGuide.t2t [hunk 2]
```diff
@@ -915,12 +915,14 @@ Also, although the application will try its best to match the native selection t
...
-Using MathPlayer 4 from Design Science, NVDA can read and interactively navigate supported mathematical content.
-This requires that MathPlayer 4 is installed on the computer.
-MathPlayer is available as a free download from the [MathPlayer information page https://info.wiris.com/mathplayer-info].
-After installing MathPlayer, restart NVDA.
-
-NVDA supports the following types of mathematical content:
+NVDA can  read and navigate mathematical content on the web and in other applications, providing access in both speech and braille. 
+However, in order for NVDA to read and interact with mathematical content, you will first need to install a Math component for NvDA.
+There are several NVDA add-ons available in the NVDA Add-on Store that provide support for math, including the [MathCAT NVDA add-on https://nsoiffer.github.io/MathCAT/] and [Access8Math https://addons.nvda-project.org/addons/access8math.en.html]. 
+Please refer to the [Add-on Store section #AddonsManager] to learn how to browse for and install available add-ons in NVDA.
```
Comment (seanbudd) [inline user_docs/en/userGuide.t2t:921]: Simplify, remove "for"

```suggestion
Please refer to the [Add-on Store section #AddonsManager] to learn how to browse and install available add-ons in NVDA.
```

File: user_docs/en/userGuide.t2t [hunk 3]
```diff
@@ -959,15 +961,28 @@ By default, the review cursor follows the system caret, so you can usually use t
...
 
-At this point, you can use MathPlayer commands such as the arrow keys to explore the expression.
+At this point, NVDA will enter Math mode, where  you can use commands such as the arrow keys to explore the expression.
 For example, you can move through the expression with the left and right arrow keys and zoom into a portion of the expression such as a fraction using the down arrow key.
-Please see the [MathPlayer documentation https://docs.wiris.com/mathplayer/en/mathplayer-user-manual.html] for further information.
 
 When you wish to return to the document, simply press the escape key.
 
+For more information on available commands and preferences for reading and navigating within math content, please refer to the documentation for your particular math component you have installed.
+- [mathCAT documentation https://nsoiffer.github.io/MathCAT/users.html]
```
Comment (seanbudd) [inline user_docs/en/userGuide.t2t:970]: capitalize M in MathCAT

```suggestion
- [MathCAT documentation https://nsoiffer.github.io/MathCAT/users.html]
```

File: user_docs/en/userGuide.t2t [hunk 4]
```diff
@@ -959,15 +961,28 @@ By default, the review cursor follows the system caret, so you can usually use t
...
+- [Access8Math documentation https://addons.nvda-project.org/addons/access8math.en.html]
+- [MathPlayer documentation https://docs.wiris.com/mathplayer/en/mathplayer-user-manual.html]
+-
+
 Sometimes mathematical content might be displayed as a button or other type of element which, when activated, can display a dialog or more information related to the formula.
 To activate the button or the element containing the formula, press ctrl+enter.
 
+++ Installing MathPlayer ++
```
Comment (seanbudd) [inline user_docs/en/userGuide.t2t:978]:
```suggestion
++ Installing MathPlayer ++[InstallingMathPlayer]
```

File: user_docs/en/userGuide.t2t [hunk 5]
```diff
@@ -959,15 +961,28 @@ By default, the review cursor follows the system caret, so you can usually use t
...
+Although it is generally recommended to use one of the newer NVDA add-ons to support math in NVDA, in certain limited scenarios MathPlayer may still be a more suitable choice, as other options may not yet support a particular language or Braille code.
```
[the remark this answers is not in the document]
Comment (seanbudd) [reply user_docs/en/userGuide.t2t:979]: @michaelDCurran @Qchristensen - thoughts on this change?

[the remark this answers is not in the document]
Comment (michaelDCurran) [reply user_docs/en/userGuide.t2t:979]: I just added "E.g." to the beginning of the second sentence.

File: user_docs/en/userGuide.t2t [hunk 6]
```diff
@@ -915,12 +915,14 @@ Also, although the application will try its best to match the native selection t
...
 + Reading Mathematical Content +[ReadingMath]
-Using MathPlayer 4 from Design Science, NVDA can read and interactively navigate supported mathematical content.
-This requires that MathPlayer 4 is installed on the computer.
-MathPlayer is available as a free download from the [MathPlayer information page https://info.wiris.com/mathplayer-info].
-After installing MathPlayer, restart NVDA.
-
-NVDA supports the following types of mathematical content:
+NVDA can read and navigate mathematical content on the web and in other applications, providing access in both speech and braille. 
+However, in order for NVDA to read and interact with mathematical content, you will first need to install a Math component for NvDA.
+There are several NVDA add-ons available in the NVDA Add-on Store that provide support for math, including the [MathCAT NVDA add-on https://nsoiffer.github.io/MathCAT/] and [Access8Math https://addons.nvda-project.org/addons/access8math.en.html].
```
Comment (michaelDCurran) [inline user_docs/en/userGuide.t2t:920]:
```suggestion
There are several NVDA add-ons available in the NVDA Add-on Store that provide support for math, including the [MathCAT NVDA add-on https://nsoiffer.github.io/MathCAT/] and [Access8Math https://github.com/tsengwoody/Access8Math]. 
```

File: user_docs/en/userGuide.t2t [hunk 7]
```diff
@@ -959,15 +961,29 @@ By default, the review cursor follows the system caret, so you can usually use t
...
-At this point, you can use MathPlayer commands such as the arrow keys to explore the expression.
+At this point, NVDA will enter Math mode, where you can use commands such as the arrow keys to explore the expression.
 For example, you can move through the expression with the left and right arrow keys and zoom into a portion of the expression such as a fraction using the down arrow key.
-Please see the [MathPlayer documentation https://docs.wiris.com/mathplayer/en/mathplayer-user-manual.html] for further information.
 
 When you wish to return to the document, simply press the escape key.
 
+For more information on available commands and preferences for reading and navigating within math content, please refer to the documentation for your particular math component you have installed.
+- [MathCAT documentation https://nsoiffer.github.io/MathCAT/users.html]
+- [Access8Math documentation https://addons.nvda-project.org/addons/access8math.en.html]
```
Comment (michaelDCurran) [inline user_docs/en/userGuide.t2t:971]:
```suggestion
- [Access8Math documentation https://github.com/tsengwoody/Access8Math]
```

**Review by seanbudd — APPROVED** [review]

**Review by Qchristensen — APPROVED** [review]
That all reads well, good work everyone.

**Comment by CyrilleB79** [comment]
> I would recommend that add-on authors move their home page URLs away from the old add-on website at this stage. Message ID: ***@***.***>

Sorry, I remember now why I have kept the link on the community website rather than a link to GitHub for my own add-ons. The GitHub's page has the same content as the English readme. However, the community website has translated content for non-English users.

For now, we can keep the link on GitHub, given the lack of visibility on the community website.
However, for add-on authors and users, a solution to have translated content is still requested.
In this message two months ago, @michaelDCurran you wrote:
> * The webhook and cron jobs will be re-enabled, again allowing add-on translations and add-on website updates to occur.

I have not seen any notification that it has happened since then. When will this happen? Since authors will update their add-ons for NVDA 2024.1 compatibility, I hope this happens before 2024.1rc1, i.e. before add-on authors are able to release stable add-ons compatible with NVDA 2024.1. Thanks.

**Comment by tsengwoody** [comment]
I am sorry for my delayed response to this discussion regarding the inclusion of Access8Math in the NVDA User Guide. It was only today, as I began updating Access8Math for compatibility with NVDA 2024.1, that I stumbled upon the updated User Guide and this discussion.
First and foremost, I would like to express my sincerest gratitude to @CyrilleB79 for advocating for Access8Math's inclusion in the User Guide. Your support for our efforts to make mathematics more accessible is deeply appreciated.
I am also immensely glad to @michaelDCurran for your openness to include Access8Math in the User Guide upon learning about it. The inclusion significantly enhances the visibility of Access8Math, benefiting users who can leverage it to its full potential. It is truly heartening to see our work acknowledged.
The mission of Access8Math has always been to offer a comprehensive and accessible solution for mathematical content. Beyond striving to provide accessible mathematical reading solution, Access8Math is equally dedicated to offering a more accessible mathematical writing solution. This  two aspects(reading/writing feature) ensures that Access8Math addresses the full spectrum of mathematical accessibility needs.
We are committed to the ongoing development of Access8Math, ensuring it remains a relevant and valuable tool for the NVDA community in future.

**Comment by tsengwoody** [comment]
I am sorry for my delayed response to this discussion regarding the inclusion of Access8Math in the NVDA User Guide. It was only today, as I began updating Access8Math for compatibility with NVDA 2024.1, that I stumbled upon the updated User Guide and this discussion.
First and foremost, I would like to express my sincerest gratitude to @CyrilleB79 for advocating for Access8Math's inclusion in the User Guide. thank your support for our efforts.
I am also immensely glad to @michaelDCurran for your openness to include Access8Math in the User Guide upon learning about it. The inclusion significantly enhances the visibility of Access8Math, benefiting users who can leverage it to its full potential. It is truly heartening to see our work acknowledged.
The mission of Access8Math has always been to offer a comprehensive and accessible solution for mathematical content. Beyond striving to provide accessible mathematical reading solution, Access8Math is equally dedicated to offering a more accessible mathematical writing solution. This two aspects ensures that Access8Math addresses the full spectrum of mathematical accessibility needs.
We are committed to the ongoing development of Access8Math, ensuring it remains a valuable tool for the NVDA community in future.

**Pull request merged**